chalk.c

```diff
@@ -15,7 +23,6 @@ void enableRawMode()
 int main()
 {
 	enableRawMode();
-
```

Comment:
Keep this extra line as a separator between the function call and the reading operation